app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -93,6 +93,13 @@
   font-size: px(40);
 }
 
+@media (max-width: 425px){
```

Comment:
We've used $mq-s now which solves the problem. Thanks, I'm not too familiar with media queries.

The 425 was totally flimsy but didn't know of a better way to do it.

The Welsh text for the button, as you can guess, is even longer :-)

Thanks